src/FEValues/common_values.jl

```diff
@@ -117,12 +142,17 @@ curl_from_gradient(∇v::SecondOrderTensor{3}) = Vec{3}((∇v[3,2] - ∇v[2,3],
 
 """
     function_value(fe_v::AbstractValues, q_point::Int, u::AbstractVector)
+    function_value(iv::InterfaceValues, q_point::Int, u::AbstractVector; use_element_a::Bool = true)
 
 Compute the value of the function in a quadrature point. `u` is a vector with values
 for the degrees of freedom. For a scalar valued function, `u` contains scalars.
 For a vector valued function, `u` can be a vector of scalars (for use of `VectorValues`)
 or `u` can be a vector of `Vec`s (for use with ScalarValues).
 
+For `InterfaceValues`, `use_element_a` determines which element to use for calculating function value. This is helpful when iterating over
```

Comment:
Probably this could be it's own docstring for `function_value(::InterfaceValues)`.